src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```

Comment:
- [ ] As it is a [requirement](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/blob/master/react_redux/math_magicians/2_Components.md) for this project " `Use class based components (All components should be class-based)` ", please kindly covert your `App.js` component which is currently a functional component to a class-based component. 

```javascript
class App extends Component {
  render() {

    return {
      <Calculator />
    };
  }
}
```

Reply:
- [ ] [REQUIRED] As recommended by the previous reviewer, kindly ensure that the `App.js` component is class-based.

```
class App extends Component {
  render() {

    return {
      <Calculator />
    };
  }
}
```

Reply:
Thanks for the comment. The components have been changed to class-based components as required